params/no_going_back.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
This one doesn't seem to be doing much.

Think about the probabilities here, how much does `no_backtracking` adjust the probabilities when you start to have several transformations like in this case of 3 pairs?

Reply:
Yup, you get better results with 1 or 2 transformations. Let me make this one a 2D transformation.

Reply:
eh, not very noticeable. I'm just removing this file.